src/js/components/help/application-updates/__snapshots__/mender-deb-package.test.js.snap

```diff
@@ -74,8 +74,7 @@ exports[`DebPackage Component renders correctly 1`] = `
         }
       }
     >
-      wget https://d1b0l86ne08fsf.cloudfront.net/master/dist-packages/debian/armhf/mender-client_master-1_armhf.deb && \\
-sudo dpkg -i --force-confdef --force-confold mender-client_master-1_armhf.deb
+      wget -q -O- https://get.mender.io/ | sudo bash
```

Comment:
Here, and everywhere else, we want to use experimental for now. Something like:
```suggestion
      wget -q -O- https://get.mender.io/ | sudo bash -s -- -c experimental
```

Reply:
https://github.com/mendersoftware/gui/pull/1111/commits/19ce832a44490aeef5fb572ddcd2cbd49c36a518